README.md

```diff
@@ -56,15 +97,164 @@ This library is a simpler interface to Pelion DM Client, making it trivial to ex
     }
     ```
 
-## Example applications
+### Example applications
+
+  To help you start quickly, please refer to the following [generic application example](https://github.com/ARMmbed/pelion-ready-example). It demonstrates how to connect to the Pelion IoT Platform service, register resources and get ready to receive a firmware update.
+
+  Also, there are a number of board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Pelion [Quick-Start](https://cloud.mbed.com/quick-start).
+
```

Comment:
IMO it's very important that we add a list or table with information of Quick-start applications, so people can use it with Mbed CLI and other tools. This is the ideal place.
It should include: Platform(s) name, Connectivity, Storage, URL to example

Reply:
Do you have that list?

Reply:
See proposal here
https://gist.github.com/MarceloSalazar/9d2c5683277d598affa215dd8020d2e4

Reply:
Now added

Reply:
If we add this to the docs, would you like these to be presented with the Import to IDE button?

Reply:
I'd suggest we keep them without the import button. Having the url will let developers review the example's doc before deciding where to import (ide or cli)